genetics-research.html

```diff
@@ -252,9 +297,37 @@ <h2 class="section__heading" id="phylogen">
                             </h2>
                             <div class="wysiwyg">
                                 <p>
-                                    Along with the tree sequence format that can store phylogenetic trees, tskit
-                                    also has methods to generate and compare topologies. Efficient access to the trees'
-                                    structure is provided by the quintuply linked tree arrays.
+                                    Tskit can be used as a phylogenetic computing library
+                                    for fast processing of very large evolutionary
+                                    trees. Methods exist to query features of trees
+                                    and traverse through their nodes, and efficient access
+                                    is also possible via arrays that provide direct
+                                    memory access to the underlying tree structure.
+                                    
+                                    Arbitrary structured or unstructured metadata can be
+                                    associated with nodes, edges, and other entities
+                                    (e.g. mutations, populations) in a tree sequence.
+                                    This provides a flexible way to associate data such
+                                    as names, geographical locations, and phenotypes
+                                    with sample genomes and their ancestors in the tree.
+                                    <!-- Add in stuff about reading in trees, when we have it --> 
+
+                                    <!-- Other tree-based methods in tskit--> 
+                                    Tskit also provides generalizable methods to
+                                    calculate statistics based on tree branch lengths,
+                                    as well as topological methods that generate,
+                                    classify, and compare tree shapes. Optimised methods
+                                    are provided that map genetic variation
+                                    parsimoniously onto a tskit tree, and other
+                                    efficient phylogenetic algorithms can be developed
+                                    in Python using the numba to achieve near-native speeds.
```

Comment:
```suggestion
                                    in Python using numba to achieve near-native speeds.
```

Maybe make this a link to the numba webpage?

Reply:
Yes, there are a number(!) of places where we could add links. I'll do this one, and we can make a second pass for the others.